### PR TITLE
Rename the knife common options page + add a missing option

### PR DESCRIPTION
**Pull request by tas50**
The name didn't bookmark or search very well.

### CODE REVIEW

File: chef_master/source/knife.rst [hunk 1]
```diff
@@ -27,7 +27,7 @@ knife is a command-line tool that provides an interface between a local chef-rep
    * - :doc:`knife_using`
      - knife runs from a management workstation and sits in-between a Chef server and an organization's infrastructure.
    * - :doc:`knife_common_options`
-     - There are many options that are available for all knife subcommands.
+     - There are many command line options that are available for all knife subcommands.
```
Comment (brewn) [inline chef_master/source/knife.rst]: The introductory paragraph states that "knife is a command-line tool that provides..." I don't think it needs to be stated that the options are command line-based.

File: chef_master/source/knife_common_options.rst [hunk 2]
```diff
@@ -1,5 +1,5 @@
 =====================================================
-Common Options
+Knife Common Command Line Options
```
Comment (brewn) [inline chef_master/source/knife_common_options.rst]: Agreed that this needs a change, however I prefer "Knife Common Options" as I don't think it's necessary to specify the command line bit.

**Comment by tas50** [comment]
@brewn rebased it down to a single commit

**Comment by brewn** [comment]
Thanks @tas50 
tenor-113939267

**Pull request merged**